### DISCUSSION TITLE
MultiSelect/TagsInput: placeholder doesn't dissapear after selections are made

### DISCUSSION THREAD

**Discussion by zuyan9** [mantinedev/mantine#6499] [Feedback]
There is a closed issue([#5288](https://github.com/mantinedev/mantine/issues/5288)) for this. The behavior is that the placeholder does not disappear after selections are made.

I'm a little confused that this is the intended behavior. It even looks weird on the official [doc](https://mantine.dev/core/multi-select/#error-state):
<img width="832" alt="image" src="https://github.com/mantinedev/mantine/assets/7322443/6b2e9cc7-7a8f-4199-b3ca-7539155acf71">

**Comment by rtivital** [comment]
Placeholder gives user a hint that more values can be entered. If you want to disable pleaceholder, set `placeholder=""`

**Reply by zuyan9** [reply]
While Select is different: the placeholder disappears after a selection is made:
<img width="835" alt="image" src="https://github.com/mantinedev/mantine/assets/7322443/7bf41e91-e577-4dad-b3b8-64359e7d966e">

Also, the word “placeholder” itself indicates this is a temporary position or content filler.

**Reply by rtivital** [reply]
You cannot select more values when Select already has a value. In MultiSelect it is possible to select as many options as there are present in the data array.

**Reply by rtivital** [reply]
Again, if you do not like this behavior, it can be changed easily with `placeholder=""`
